packages/loot-core/src/server/main.ts

```diff
@@ -1162,6 +1162,45 @@ handlers['accounts-sync'] = async function ({ id }) {
   return { errors, newTransactions, matchedTransactions, updatedAccounts };
 };
 
+handlers['secret-set'] = async function ({ name, value }) {
+  let userToken = await asyncStorage.getItem('user-token');
+
+  if (userToken) {
+    try {
+      return await post(
+        getServer().BASE_SERVER + '/secret',
+        {
+          name,
+          value,
+        },
+        {
+          'X-ACTUAL-TOKEN': userToken,
+        },
+      );
+    } catch (error) {
+      console.error(error);
+      return { error: 'failed' };
+    }
+  }
+  return { error: 'unauthorized' };
+};
+
+handlers['secret-get'] = async function (name) {
```

Comment:
Maybe `secret-check` since you’re not reading the value? (And then it could return a boolean to say whether the value exists or not)

```suggestion
handlers['secret-check'] = async function (name) {
```

Reply:
Also, this looks not to be used at the moment — is that intentional?

Reply:
https://github.com/actualbudget/actual/pull/968/commits/8553082d9d8e6a7048ed63ea04c4e1142242d975

Yea, it's intentional. We don't actually need it for nordigen since it already has `nordigen-status` check. But I'm adding it nonetheless since it will be useful for Plaid or other things needing secrets.